src/sm/Elements/Bars/truss3d.C

```diff
@@ -245,6 +245,26 @@ Truss3d::computeStressVector(FloatArray &answer, const FloatArray &strain, Gauss
     answer = this->giveStructuralCrossSection()->giveRealStress_1d(strain, gp, tStep);
 }
 
+int
+Truss3d::giveIPValue(FloatArray &answer, GaussPoint *gp, InternalStateType type, TimeStep *tStep)
+{
+    if ( type == IST_BeamForceMomentTensor ) {
+        FloatArray stress, strain;
+        double area;
+        this->computeStrainVector(strain, gp, tStep);
+        this->computeStressVector(stress, strain, gp, tStep);
+        area = this->giveCrossSection()->give(CS_Area, gp);
+        answer.resize(1);
```

Comment:
The IST_BeamForceMomentTensor should be of size 6 (see vtkbaseexportmodule.C:558, for example)